src/client/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -405,5 +405,9 @@ public static string CertMustHavePrivateKey(string certificateName)
         public const string TenantOverrideNonAad = "WithTenantId can only be used when an AAD authority is specified at the application level.";
 
         public const string RegionalAndAuthorityOverride = "You configured WithAuthority at the request level, and also WithAzureRegion. This is not supported when the environment changes from application to request. Use WithTenantId at the request level instead.";
+
+        public const string OboCacheKeyAlreadyInCache = "The token cache already contains a token with an OBO cache key that matches the longRunningProcessSessionKey passed into IConfidentialClientApplication.AcquireTokenInLongRunningProcess method. Call IConfidentialClientApplication.InitiateLongRunningProcessInWebApi method with a new longRunningProcessSessionKey that does not exist in the token cache or call IConfidentialClientApplication.AcquireTokenInLongRunningProcess method with an already used longRunningProcessSessionKey.";

```

Comment:
>OboCacheKeyAlreadyInCache [](http://example.com/codeflow?start=28&length=25)

Might be a good idea for supportability to add the aka.ms link here too. Sometimes people run into issues and dont know where to look in the code to figure out what to do. We also add these links for other messages